File: demo/src/main/java/br/com/sample/data/SuggestionProvider.java
package br.com.sample.data;

import android.content.SearchRecentSuggestionsProvider;

public class SuggestionProvider extends SearchRecentSuggestionsProvider {

    public final static String AUTHORITY = "br.com.stickyindexsample.dao.SuggestionProvider";
    public final static int MODE = DATABASE_MODE_QUERIES;

    public SuggestionProvider() {
        setupSuggestions(AUTHORITY, MODE);
    }
}

File: demo/src/main/java/br/com/stickindex/sample/data/SuggestionProvider.kt
package br.com.stickindex.sample.data

import android.content.SearchRecentSuggestionsProvider

class SuggestionProvider : SearchRecentSuggestionsProvider() {
    init {
        setupSuggestions(SuggestionProvider::class.java.simpleName!!, DATABASE_MODE_QUERIES)
    }
}